### ISSUE TITLE
ESXI

### ISSUE THREAD

**Issue by oskarstr** [Leoyzen/KVM-Opencore#16]
Would you consider putting together a guide on how to make this work on ESXI with full gpu passthrough?

This package right now doesn't work on esxi - black screen. Gpu passthrough with supported gpu otherwise doesn't get hw acceleration.

**Comment by Ran-Xing** [comment]
+1

**Comment by thenickdude** [comment]
No, I do not see any reason to add support for closed source hosts, especially when getting this working on Open Source hosts has frequently required source level patches to succeed.

Closed source vendors are free to implement this themselves, but I will not not.
